modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/exp/agg/AccumulatorsFactory.java

```diff
@@ -267,11 +265,13 @@ private final class AccumulatorWrapperImpl implements AccumulatorWrapper<RowT> {
         private final boolean distinct;
 
         AccumulatorWrapperImpl(
+                RowHandler<RowT> handler,
```

Comment:
If captured context is no longer used, can this class be static?

Reply:
good point, fixed